src/core_ocean/shared/mpas_ocn_tendency.F

```diff
@@ -868,6 +869,22 @@ subroutine ocn_tend_tracer(tendPool, statePool, forcingPool, &
 
          endif ! MacroMolecules
 
+         !
+         ! compute suspended sediment source-sink tendencies and net surface fluxes
+         ! NOTE: must be called before ocn_tracer_surface_flux_tend
+         !
+         if ( groupName == 'sedimentTracers' ) then
+            call ocn_tracer_sediment_compute(tracerGroup, &
+                                     nTracersGroup,forcingPool,  &
+                                     nCellsOwned, maxLevelCell, &
```

Comment:
We recently put in some code that allows for inactive top cells, so `minLevelCell` is needed here and in the subroutine.

Reply:
Thanks. I will add `minLevelCell`. The whole routine currently works as a placeholder, and I will modify it accordingly as the development work goes on.